documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,90 @@
+      3. Sending corrections of competition results to the WCA Results Team.
+      4. Advising other competition officials when necessary.
+      5. Deciding about disqualifying competitors during the competition.
+      6. Generating, providing, and maintaining the security of the scramble sequences ([WCA Regulation 4b2)](wca{regulations/#4b2})).
+      7. Making a copy of the WCA Regulations available at the competition.
+   2. The WCA Delegate may delegate some of their responsibilities to organizers or other officials.
+
+If WCA Regulations are not correctly observed during a competition, results may be modified at the discretion of the WCA Delegate, WCA Regulations Committee, WCA Results Team, WCA Disciplinary Committee, or the WCA Board.
+
+### Announcement Criteria
+The WCA Delegate must submit the following information when requesting approval of a competition:
+
+1. Competition name, nickname, and ID
+   1. The competition name must abide by the [WCA Competition Name Policy](wca{documents/policies/Competition Name.pdf}).
+   2. The competition nickname and ID must resemble the competition name.
+      1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters).
+2. Location(s)
+   1. A location consists of the country, city, and venue.
+   2. The location should also include subregions of the country that are common to include with addresses (e.g. the state a U.S. city is located in).
+3. WCA Delegate(s)
+4. Organizer(s)
+   1. Organizers can be specific persons and/or organizations.
+5. Competitor limit
+   1. The competitor limit can be on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date.
+      1. Qualification for an event must be based only on results from the same event.
+   2. If the competitor limit for a competition or an event is reached before the competition, the organization team must not approve any additional registrations for the competition or the event.
+   3. The competitor limit is per competition. Per day competitor limits are not allowed.
+6. Registration requirements
+   1. The registration requirements must be clear so that any competitor from anywhere in the world knows how to and has the ability to register.
+      1. The registration requirements must be available in English.
+      2. In addition to the English version, the registration requirements can be available in any other language and should be available in the local language.
+   2. The registration period(s) must be a clearly defined time range with a start and end time and date.
+      1. The first registration period must have a start time and date at least 48 hours after the competition is announced.
```

Comment:
```suggestion
       1. The first registration period must have a start time and date at least 48 hours after the competition is announced.
          1. The competition's Delegate(s) and/or organizers may register for the competition prior to the first registration period.
```
Ideally, this would be automatic, but until then, I think allowing this manually (and then just accepting the registrations once the registration period opens) is best.

Reply:
For the record, wcat is strictly against this.

Reply:
Against automatic registration of Delegates and/or organizers?

Reply:
Allowing them to register before everyone else

Reply:
FTR: the Board consulted the WEC about this before making a final decision.

Reply:
@AlbertoPdRF, could you explain what that final decision was that the Board and the WEC came do? Could you also update https://github.com/thewca/worldcubeassociation.org/issues/1692 with the results of that decision? Thanks!

Reply:
WCAT is against this and WEC is in favor. We are bringing the discussion to Leaders + Seniors now.

Reply:
After consultation with Leaders and Seniors, the Board has decided to allow Delegates and/or organizers to pre-register for their competitions, although their registrations must only be approved when registration opens.